tests/client/core/TestAddonManager.js

```diff
@@ -36,6 +36,16 @@ describe('addonManager', () => {
     };
   });
 
+  describe('hasAddonManager', () => {
+    it('does if mozAddonManager is in navigator', () => {
```

Comment:
this would be less awkward to read for me if it was: `it('returns true if mozAddonManager...')`.